tests/loggers/test_wandb_logger.py

```diff
@@ -259,32 +269,35 @@ def test_wandb_is_pickleable_when_disabled(dummy_state: State):
 
 @pytest.mark.world_size(2)
 @pytest.mark.parametrize('rank_zero_only', [True, False])
-@pytest.mark.skip('This test needs to be refactored to use a Mock API interface.')
-def test_wandb_artifacts(rank_zero_only: bool, tmp_path: pathlib.Path, dummy_state: State):
-    """Test that wandb artifacts logged on rank zero are accessible by all ranks."""
-    pytest.importorskip('wandb', reason='wandb is optional')
+def test_wandb_artifacts(monkeypatch: MonkeyPatch, rank_zero_only: bool, tmp_path: pathlib.Path, dummy_state: State):
+    """Test that wandb artifacts logged on rank zero are accessible."""
+    from wandb import Api
+    os.environ['WANDB_MODE'] = 'offline'
```

Comment:
I think at the end of this tests you want to set the env var back to whatever it was initially?